ClemBot.Bot/Dockerfile

```diff
@@ -7,21 +7,19 @@ ENV PYTHONDONTWRITEBYTECODE 1
 # Turns off buffering for easier container logging
 ENV PYTHONUNBUFFERED 1
 
-# Set up and activate virtual environment
-ENV VIRTUAL_ENV "/venv"
-
 RUN apt-get update && apt-get install -y git
 
-RUN python -m venv $VIRTUAL_ENV
-ENV PATH "$VIRTUAL_ENV/bin:$PATH"
+# Install poetry
+RUN python -m pip install poetry
 
-# Install pip requirements
-ADD requirements.txt .
-RUN python -m pip install -r requirements.txt
+# Install dependencies with Poetry
+ADD pyproject.toml .
+ADD poetry.lock .
+RUN python -m poetry install
 
 WORKDIR /ClemBot.Bot
 ADD . /ClemBot.Bot
 
 
 # During debugging, this entry point will be overridden. For more information, please refer to https://aka.ms/vscode-docker-python-debug
-CMD ["python","-m", "bot"]
+CMD ["python", "-m", "poetry", "run", "python", "-m", "bot"]
```

Comment:
Is the command to run really "python -m poetry run python -m bot"?

Reply:
Well I'm not sure how this particular distro and docker play with poetry, I figured better safe than sorry, this should work. If poetry is installed like it is on my particular machine, it would be possible to do `poetry run python -m bot`, but again I was unsure.

Reply:
It's  docker image, not your machine. What did you run in docker to test it